compair/static/modules/comparison/comparison-form-partial.html

```diff
@@ -151,22 +151,22 @@ <h3 class="text-center"><compair-logo type="'scale'"></compair-logo> Answer {{se
                 <fieldset>
                     <legend>Peer Feedback</legend>
                     <div class="col-md-6">
-                        <label class="cke-label required-star">
+                        <label class="cke-label {{commentRequired ? ' required-star' : ''}}">
                             Your private feedback for Answer {{firstAnsNum}}'s author
                             <i class="fa fa-arrow-down"></i>
                         </label>
                         <textarea
-                            ng-model="answer1.comment.content" rows="8" ckeditor="editor1Options" required>
+                            ng-model="answer1.comment.content" rows="8" ckeditor="editor1Options" ng-required="{{commentRequired}}">
```

Comment:
maybe just use `canManageAssignment` as the condition instead of `commentRequired`